src/java_bytecode/java_bytecode_language.h

```diff
@@ -37,7 +37,8 @@ Author: Daniel Kroening, kroening@kroening.com
   "(java-cp-include-files):"                                                   \
   "(lazy-methods)"                                                             \
   "(lazy-methods-extra-entry-point):"                                          \
-  "(java-load-class):"
+  "(java-load-class):"                                                         \
+  "(java-no-load-class):"
```

Comment:
It would be useful to document (in the help string) what that method does and how it interacts with the other options.

Reply:
Help for both `java-load-class` and `java-no-load-class` is located in test-gen. I'll create another PR that moves them here.